### PR TITLE
Wallet connect fallback

### LINKED ISSUE #955: WEB/HUBS - if you try to do an action that requires a wallet the sign WalletConnectModal should be shown

### PR DESCRIPTION
**Pull request by ikjudd**
- Closes #955

### CODE REVIEW

**Comment by ikjudd** [comment]
**WEB**
- [ ] Wallet Modal displays when collecting a free Release
- [ ] Wallet Modal displays when purchasing a Release
- [ ] Wallet Modal displays when creating a Buy Offer
- [ ]  Wallet Modal displays when creating a Sell Offer
- [ ]  Wallet Modal displays when buying from Exchange
- [ ] Wallet Modal displays when attempting to Repost a Release

**HUBS**
- [ ] Wallet Modal displays when collecting a free Release
- [ ] Wallet Modal displays when purchasing a Release
- [ ] Wallet Modal displays when creating a Buy Offer
- [ ]  Wallet Modal displays when creating a Sell Offer
- [ ]  Wallet Modal displays when buying from Exchange
- [ ] Wallet Modal displays when attempting to Repost a Release

File: js/sdk/src/index.js [hunk 1]
```diff
@@ -34,4 +34,6 @@ export { default as WalletConnectModal } from './components/WalletConnectModal'
 export { default as WalletButton } from './components/WalletButton'
 export { default as NoSolWarning } from './components/NoSolWarning'
 export { default as Swap } from './components/Swap'
-export { default as IdentityVerification } from './components/IdentityVerification'
+export { default as IndentityVerification } from './components/IndentityVerification'
```
Comment (ejf89) [inline js/sdk/src/index.js:37]: typo

File: js/web/components/ReleaseCard.js [hunk 2]
```diff
@@ -100,7 +99,7 @@ const ReleaseCard = (props) => {
                     updateTrack(releasePubkey, true, true)
                   }
                 }}
-                sx={{ height: '22px', width: '28px' }}
+                sx={{ height: '22px', width: '22px' }}
```
Comment (ejf89) [inline js/web/components/ReleaseCard.js:102]: why we changing this?

**Review by ejf89 — CHANGES_REQUESTED** [review]
plz fix type and cta placement in `releaseCard`

**Pull request merged**